src/www/ui/api/Controllers/InfoController.php

```diff
@@ -60,6 +60,19 @@ public function getInfo($request, $response, $args)
     foreach ($yamlDocArray["security"] as $secMethod) {
       $security[] = key($secMethod);
     }
+    global $SysConf;
+    $fossologyVersion = "";
+    $buildDate = "";
+    $commitHash = "";
+    $commitDate = "";
+    $branchName = "";
+    if (array_key_exists('BUILD', $SysConf)) {
+      $fossologyVersion = $SysConf['BUILD']['VERSION'];
+      $buildDate = date(DATE_ISO8601, strtotime($SysConf['BUILD']['BUILD_DATE']));
```

Comment:
A faster way will be `date_create_from_format('Y/m/d H:i e', $SysConf['BUILD']['BUILD_DATE'])->format('c')` as `strtotime` has to guess the format whereas you will be telling it explicitly.

Reply:
There is one side-effect of PHP that it detects `IST` as Istanbul instead of India. So effective timezone becomes `+02:00` instead of `+05:30`.

Reply:
@GMishx Updated it